app/src/main/kotlin/com/ayatk/biblio/ui/ranking/RankingActivity.kt

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2016-2018 ayatk.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ayatk.biblio.ui.ranking
+
+import android.content.Context
+import android.content.Intent
+import android.databinding.DataBindingUtil
+import android.os.Bundle
+import android.support.v4.app.Fragment
+import android.support.v4.app.FragmentManager
+import android.support.v4.app.FragmentPagerAdapter
+import com.ayatk.biblio.R
+import com.ayatk.biblio.data.narou.entity.enums.RankingType
+import com.ayatk.biblio.databinding.ActivityRankingBinding
+import com.ayatk.biblio.ui.util.initBackToolbar
+import com.ayatk.biblio.util.ext.extraOf
+import dagger.android.support.DaggerAppCompatActivity
+
+class RankingActivity : DaggerAppCompatActivity() {
+
+  private val binding: ActivityRankingBinding by lazy {
+    DataBindingUtil.setContentView<ActivityRankingBinding>(this, R.layout.activity_ranking)
+  }
+
+  private val rankingType: RankingType by lazy {
+    intent?.getSerializableExtra(EXTRA_RANKING_TYPE)!! as RankingType
+  }
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+
+    initBackToolbar(this, binding.toolbar)
+
+    binding.rankingViewPager.apply {
+      adapter = RankingPagerAdapter(supportFragmentManager)
+      pageMargin = 16
```

Comment:
**[detekt]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.